newIDE/app/src/EventsFunctionsExtensionEditor/index.js

```diff
@@ -301,6 +301,48 @@ export default class EventsFunctionsExtensionEditor extends React.Component<
     done(true);
   };
 
+  _makeMoveFreeEventsParameter = (i18n: I18nType) => (
+    eventsFunction: gdEventsFunction,
+    oldIndex: number,
+    newIndex: number,
+    done: boolean => void
+  ) => {
+    //TODO add verifications and warning box
```

Comment:
Let's replace this TODO by `// Don't ask for user confirmation as this change is easy to revert.`